app/src/main/java/com/infomaniak/mail/data/models/calendar/CalendarEventResponse.kt

```diff
@@ -54,7 +53,7 @@ class CalendarEventResponse() : EmbeddedRealmObject {
 
     val calendarEvent get() = userStoredEvent ?: attachmentEvent
 
-    val isCanceled get() = isUserStoredEventDeleted || attachmentEventMethod == AttachmentEventMethod.CANCEL
+    val isCanceled get() = calendarEvent?.status == CalendarEventStatus.CANCELLED
```

Comment:
👍 We must check the status of an event, and only the status, to know if it has been canceled of not